src/Input/input.scss

```diff
@@ -0,0 +1,50 @@
+@import '../utils/theme/variables.scss';
+$height: 28px !default;
+$padding-h: 12px !default;
+$font-size: 14px !default;
+$font-weight: $brc-regular !default;
+$placeholder-color: $brc-coolGrey !default;
+$color: $brc-darkGrey !default;
+$border-color: $brc-silver !default;
+$color-focus: $brc-darkGrey !default;
+$border-color-focus: $brc-waterBlue !default;
+$disabled-opacity: .6 !default;
```

Comment:
this should be named `opacity-disabled` :P

Reply:
fixed!